src/kudu/tserver/simple_tablet_manager.cc

```diff
@@ -109,7 +109,7 @@ using pb_util::SecureShortDebugString;
 
 namespace tserver {
 
-const std::string TSTabletManager::kSysCatalogTabletId("00000000000000000000000000000000");
+std::string TSTabletManager::kSysCatalogTabletId("00000000000000000000000000000000");
```

Comment:
Can we make this an instance variable instead of a global? This was global previously because it was static const